src/tests_common.rs

```diff
@@ -199,6 +200,7 @@ pub struct PrefixExpansion {
     pub expanded_length: usize,
 }
 
+#[cfg(feature = "std")]
```

Comment:
hmmmm, I split on this, but maybe we could move the `#[cfg(feature = "std")]` to just the block where it checks `uniq_indices`. 

That way we would at least be able to use more tests which use `generate_key_with_prefix`